Refine Configure_https.md

jeffweilee opened the pull request:
Rebase PR #4279
Close Issue #4258 

Refine Configure_https.md so that HTTPS can work and comply with Subject Alternative Name (SAN) and x509 v3 extension requirement. The document outline is as follows:

Getting Certificate Authority
Getting Server Certificate
Getting Client Certificate
Configuration and Installation
Troubleshooting

clouderati commented:
Hi Jeff – thanks for the contribution. Looks good, but I think you `pick`ed a few commits from other contributors when rebasing.

Can you do a `git rebase -i HEAD~13` (I believe `13` should work here; you can go back further if you're certain to `drop` appropriately) and make sure you:
* `squash` (s) your multiple commits
* `drop` (d) commits by other contributors

I'll merge as soon as that's done. :smile: Feel free to ping me on Slack if you have any questions or I can help in any way.

jeffweilee commented:
Hi @clouderati, thanks for your reminder, I squashed and dropped irrelevant commits into 464fa37. Please kindly let me know if anything wrong, thanks.
